boa3_test/examples/README.md

```diff
@@ -0,0 +1,44 @@
+# Examples
+
+This folder has some examples of smart contracts that are used throughout the blockchain. It's also recommended checking
+the [tests](https://github.com/CityOfZion/neo3-boa/tree/development/boa3_test/tests/examples_tests), since they have 
+comments describing why the methods are being called, and their expected behaviour.
+
+## AMM
+
+The AMM (Automated Market Maker) is a smart contract with the goal to automatize token trading. Instead of buying from 
+and selling to other users, there is a liquidity pool that stores two tokens and if the user wants to buy tokens from 
+there, they can.
+The example was mainly based on [Uniswap V2](https://uniswap.org/blog/uniswap-v2/)
+
```

Comment:
The AMM (Automated Market Maker) is a smart contract that automates the process of token trading. This example was mainly based on [Uniswap V2](https://uniswap.org/blog/uniswap-v2/), and sowcases one of the simplest implementations of blockchain AMMs.
It establishes a liquidity pool of two specific tokens, that are kept at a fixed ratio in the pool. Users can provide liquidity to the pool by sending amounts of the accepted token, and receiving a liquidity token that represents their participation in the pool. Other users can then swap the tokens directly one for the other. Liquidity providers receive a part of the fees as incentive to keep their tokens in the pool. The relative prices of the token pair varies according to the expected fixed ratio, and if they fluctuate away from prices on other markets, it creates an incentive for users to make a profit on this difference, thus returning balance to the pool.